lib/forspell/cli.rb

```diff
@@ -38,28 +38,27 @@ def initialize options
 
     def call
       init_options
+      init_reporter
       create_files_list
       init_speller
-      init_reporter
       run
     end
 
     private
 
     def create_files_list
       @files = FileList.new(paths: @opts.arguments, exclude_paths: @opts[:exclude_paths])
+    rescue Forspell::FileList::PathLoadError => path
+      @reporter.path_load_error path
+      exit ERROR_CODE
     end
 
     def init_options
       @options += File.read(CONFIG_PATH).tr("\n", ' ').split(' ') if File.exist?(CONFIG_PATH)
 
       @opts = Slop.parse(@options, &DEFINITIONS)
 
-      if @opts.arguments.empty?
-        puts 'Usage: forspell paths to check [options]'
-        puts 'Type --help for more info'
-        exit(ERROR_CODE)
-      end
+      @opts.arguments << '.' if @opts.arguments.empty?
```

Comment:
Maybe then it makes sense to start from the information line with list of pathes really checked (to avoid any confusion). Though it is not mandatory :)

Reply:
I'd skip it, should add default behavior to readme though.